src/main/java/graphql/Directives.java

```diff
@@ -124,4 +127,14 @@ private static Description createDescription(String s) {
     }
 
 
+    private final static Set<String> SPECIFIED_DIRECTIVES = ImmutableSet.of(
```

Comment:
we have the same methods/logic already in DirectiveInfo I believe.

Reply:
nice